incubator/mongodb/README.md

```diff
@@ -0,0 +1,192 @@
+# MongoDB Helm Chart
+
+## Prerequisites Details
+* Kubernetes 1.3 with alpha APIs enable
+* PV support on the underlying infrastructure
+
+## PetSet Details
+* http://kubernetes.io/docs/user-guide/petset/
+
+## PetSet Caveats
+* http://kubernetes.io/docs/user-guide/petset/#alpha-limitations
+
+# TODO
+* Set up authorization between replicaset peers.
+* Set up sharding.
+
+## Chart Details
+
+This chart implements a dynamically scalable mongoDB replicaset 
```

Comment:
Should we call it a "Mongo Replicaset" ... and Cassandra uses the term "nodes".... 😂